cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp

```diff
@@ -122,7 +123,7 @@
             <input type="checkbox" name="rememberMe" id="rememberMe" value="true" checked tabindex="5"/>
             <label for="rememberMe"><spring:message code="screen.rememberme.checkbox.title"/></label>
             <spring:eval var="forgotPasswordUrl" expression="@casProperties.getProperty('osf.forgotPassword.url')"/>
-            <a id="forgot-password" class='need-help' href="${forgotPasswordUrl}" title="<spring:message code="logo.title" />"><spring:message code="screen.general.link.forgotPassword"/></a>
```

Comment:
`title="<spring:message code="logo.title"` is wrong and redundant where `logo.title=OSF Sign In`. Thus removed. This surfaced in this PR since the "Not your institution?" link originates from this tag.